src/components/Dashboard/dashboard.less

```diff
@@ -348,6 +358,13 @@
     color: #ffffff;
     margin-top: 6px;
     display: inline-block;
+
+    &.listItemOutcome {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      width: 76%;
```

Comment:
Hmm, does this work with all screen sizes?

Reply:
@andre-meyer It works the same as our interface, not ready for small screens